src/components/HomePage/MainView.js

```diff
@@ -37,13 +53,18 @@ const MainView = ({ scrollToProductDescription, showSearchResults }) => {
             </CenteredComponent>
             <div className={classes.infoBox}>
                 <InfoBox size={useMobile() ? "small" : "normal"}>
-                    Your next oportunity is out there. Use the search bar to find it!
+                    Your next opportunity is out there. Use the search bar to find it!
                 </InfoBox>
             </div>
             <div className={classes.showMoreBtn}>
-                <ShowMoreButton
-                    onClick={scrollToProductDescription}
-                />
+                {
+                    scrollPosition < switchScrollButtonPosition ?
```

Comment:
Why the magical constant 450? Does that work on all screens/viewports? It might be a better idea to use a `ref`on the button div (which will give you access to the DOM element at ref.current, once the component mounts) and then you can use something like this (https://stackoverflow.com/questions/8070639/find-elements-position-in-browser-scroll/55963021)